### PR TITLE
chore: remove unused taggers field

### PR DESCRIPTION
**Pull request by cuishuang**
Fixes: #nnn 
**Related**: _Relevant tracking issues, for context_
**Merge before/after**: _Dependent or prerequisite PRs_

**Description**

**User facing changes (remove if N/A)**

**Follow-up Work (remove if N/A)**

remove no-used taggers field

### CODE REVIEW

**Review by plumpy — CHANGES_REQUESTED** [review]
Interesting, I was wondering why this wasn't a problem before, and it looks like it's because the `taggers` field is never actually used. Can you just delete it instead?

Thanks for the PR!

**Comment by cuishuang** [comment]
> Interesting, I was wondering why this wasn't a problem before, and it looks like it's because the `taggers` field is never actually used. Can you just delete it instead?
> 
> Thanks for the PR!

Of course. Please review again.

File: pkg/skaffold/tag/tagger_mux.go [hunk 1]
```diff
@@ -41,18 +41,16 @@ func (t *TaggerMux) GenerateTag(ctx context.Context, image latest.Artifact) (str
...
 		if err != nil {
 			return nil, fmt.Errorf("creating tagger: %w", err)
 		}
-		sl = append(sl, t)
 		for _, a := range p.Build.Artifacts {
 			m[a.ImageName] = t
 		}
 	}
-	return &TaggerMux{taggers: sl, byImageName: m}, nil
+	return &TaggerMux{byImageName: m}, nil
```
Comment (plumpy) [inline pkg/skaffold/tag/tagger_mux.go:53]: Can you also remove the field from the TaggerMux struct (on line 29)?

Thanks!

Comment (cuishuang) [reply pkg/skaffold/tag/tagger_mux.go:52]: My pleasure.

Please review again.

**Comment by plumpy** [comment]
Thanks for sending the PR, we appreciate it!

**Comment by cuishuang** [comment]
> Thanks for sending the PR, we appreciate it!

I am very happy to do this.

**Pull request merged**